worlds/messenger/docs/en_The Messenger.md

```diff
@@ -48,25 +48,23 @@ for it. The groups you can use for The Messenger are:
 ## Other changes
 
 * The player can return to the Tower of Time HQ at any point by selecting the button from the options menu
-    * This can cause issues if used at specific times. Current known:
-        * During Boss fights
-        * After Courage Note collection (Corrupted Future chase)
-            * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
-is recommended to quit to title and reload the save
+  * This can cause issues if used at specific times. Current known:
+    * During Boss fights
+      * After Courage Note collection (Corrupted Future chase)
+        * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
+        is recommended to quit to title and reload the save
 * After reaching ninja village a teleport option is added to the menu to reach it quickly
 * Toggle Windmill Shuriken button is added to option menu once the item is received
 
 ## Currently known issues
 * Necro cutscene will sometimes not play correctly, but will still reward the item
 * Ruxxtin Coffin cutscene will sometimes not play correctly, but will still reward the item
-* If you receive the Fairy Bottle while in Quillshroom Marsh, The Decurse Queen cutscene will not play
+* If you receive the Fairy Bottle while in Quillshroom Marsh, The Decurse Queen cutscene will not play. You can exit
+to Searing Crags and re-enter to get it to play correctly.
```

Comment:
```suggestion
  to Searing Crags and re-enter to get it to play correctly.
```

Reply:
wait but none of the other ones are indented, this is inconsistent.

Reply:
oh, should've rejected the original PR then. Text should be aligned in lists.